pmm-app/src/pmm-qan/panel/components/ManageColumns/ManageColumns.tsx

```diff
@@ -60,15 +66,19 @@ export const ManageColumns = (props) => {
       placement="topLeft"
     >
       <div>
-        <span className={styles.placeholder}>
+        <span className={`${styles.placeholder} ${styles.placeholderPadding}`}>
```

Comment:
Better leverage a `cx` function from emotion to join classes (see example below)